Sources/MuxUploadSDK/Upload/ChunkWorker.swift

```diff
@@ -170,7 +170,15 @@ fileprivate actor ChunkActor {
     let urlSession: URLSession
     
     func upload() async throws -> URLResponse {
-        let contentRangeValue = "bytes \(chunk.startByte)-\(chunk.endByte - 1)/\(chunk.totalFileSize)"
+        let startByte = "\(chunk.startByte)"
+        let endByte: String
+        if chunk.endByte == 0 {
```

Comment:
It's sort of odd that we get a `Chunk` with size 0. That would imply `ChunkedFile` is improperly returning a `Chunk` for some reason. I'm worried there's a deeper issue here

Reply:
Just saw @dylanjha's comment, and this could also be an invalid file. It might be worth investigating SDK behavior when the file is empty/not sane.

(also Dylan just fyi you can drag-and-drop video files onto the SImulator window and they'll be saved to the camera roll)

Reply:
@daytime-em agree. if the uploaded content that can reproduce this case still works (ingests, generates a valid mezz, can be played, etc.), I'd recommend still merging this to avoid crashes and doing a fast follow on investigating the 0 byte size.

Reply:
Important to note here that as long as the SDK handles chunk request failures to GCP gracefully, the SDK baseline behavior here becomes reasonable.

It would still be good to see if the specific Simulator issue can be addressed in a future PR